src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -40,7 +41,7 @@ public MovieSessionController(MovieSessionService movieSessionService,
     }
 
     @PostMapping
-    public MovieSessionResponseDto add(@RequestBody MovieSessionRequestDto requestDto) {
+    public MovieSessionResponseDto add(@RequestBody @Valid MovieSessionRequestDto requestDto) {
```

Comment:
Let's also validate movie session request parameters in `update() `method 
(PS Can't add comments to unedited code lines, so leave this comment here)